svc/pkg/servers/db/servers/migrations/20240501133910_init.up.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE servers (
+	game_id UUID NOT NULL,
+	datacenter_id UUID NOT NULL,
+	-- The server will be locked to a certain cluster, but a game might change
+	-- clusters, and therefore the server will be moved to a new cluster.
+	cluster_id UUID NOT NULL,
+	-- This represents a map<string, json>
+	metadata JSONB NOT NULL,
+	resources_cpu_millicores INT NOT NULL,
+	resources_memory_mib INT NOT NULL,
+	kill_timeout_ms INT NOT NULL,
+	
+	create_ts INT NOT NULL,
+	-- When the server was marked to be deleted by Rivet
+	destroy_ts INT,
+
+	INDEX (game_id),
+
+	server_id UUID PRIMARY KEY
```

Comment:
nit: this is werid to put last. i see why you tried this, but `PRIMARY KEY (...)` (altering the table itself by setting the primary key) is very different than `xxxx XXXX PRIMARY KEY` (defining a column)